src/components/breedingModal.html

```diff
@@ -58,7 +58,7 @@ <h5 class="modal-title" id="breedingModalLabel">Day Care</h5>
                                 <label>Regions</label><br/>
                                 <div class="dropdown show">
                                     <button type="button" class="text-left custom-select col-12 btn" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" data-bind="text :
-                                        BreedingFilters.region.value() == 2 ** (player.highestRegion() + 1) - 1 ?
+                                        BreedingFilters.region.value() || BreedingFilters.region.value() == 2 ** (player.highestRegion() + 1) - 1 ?
                                             'All' :
                                             (BreedingFilters.region.value() ?
```

Comment:
Changing this null check to `BreedingFilters.region.value() > 0` will stop `-2` from being truthy as well.